CONTRIBUTING.md

```diff
@@ -21,56 +21,56 @@ new to this style of development, you'll want to read over our
 You may also/instead be interested in contributing to an
 [astropy affiliated package](http://www.astropy.org/affiliated/).
 Affiliated packages are astronomy-related software packages that are not a part
-of the astropy core package, but build on it for more specialized applications
-and follow the astropy guidelines for reuse, interoperability, and interfacing.
+of the Astropy core package, but build on it for more specialized applications
```

Comment:
astropy core package is usually refers to astropy the package and usually try to we keep it lowercase/highlighted (whatever else we do in the document with namespaces, class names, etc.)

Reply:
Ah, ok. I didn't realize there were instances where Astropy referred to the core package and was not in backticks. I may need help auditing where it should be used as the proper noun in text and where it should be lowercase (and in backticks?) as the package name.